index.bs

```diff
@@ -855,28 +883,29 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     <dl class="switch">
```

Comment:
I'm thinking this is a "go" given @dominc's thoughts in this comment on issue #472: https://github.com/w3c/webauthn/issues/472#issuecomment-309871552 and other examples of how algs are written, e.g. [in credman](https://w3c.github.io/webappsec-credential-management/#algorithm-request).